cpp/velox/shuffle/VeloxSplitter.cc

```diff
@@ -64,6 +64,18 @@ std::string __m128i_toString(const __m128i var) {
 }
 #endif
 
+namespace {
+
+bool VectorHasNull(const velox::VectorPtr& vp) {
+  const auto& nulls = vp->nulls();
+  if (!nulls) {
+    return false;
+  }
+  return vp->countNulls(nulls, vp->size()) != 0;
```

Comment:
Can we use vp->mayHaveNulls

Reply:
> Can we use vp->mayHaveNulls() && vp->isNullAt(i)

here we need to check whether this column has null, so if use `vp->isNullAt(i)`, we must call this in a loop whose `i` iterate from 0 to row.